### DISCUSSION TITLE
Smoke doesn't work after loading or restarting the beatmap

### DISCUSSION THREAD

**Discussion by Nattale** [ppy/osu#22253] [Q&A]
### Type

Game behaviour

### Bug description

In stable you can hold the Smoke button while it was "loading" or "restarting" to still put smoke on the screen, In lazer you can't do that, i dont know if its intentional but its there

### Version

2022.1228.0

### Logs

[input.log](https://github.com/ppy/osu/files/10433055/input.log)
[network.log](https://github.com/ppy/osu/files/10433057/network.log)
[performance.log](https://github.com/ppy/osu/files/10433058/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10433059/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10433060/updater.log)
[database.log](https://github.com/ppy/osu/files/10433061/database.log)

**Comment by peppy** [comment]
Based on how smoke is implemented, this would be a tough ask.

We'll have to see how many users relied on this.

**Reply by Inconsist** [reply]
Maybe poll it? personally I'd agree that its not necessary but with a poll thered be more evidence behind it.

**Reply by peppy** [reply]
It's not important enough to poll. Will wait for more people to complain.
